docs/env.md

```diff
@@ -14,6 +14,7 @@ with a non-standard filesystem layout.
 * `FWUPD_DEVICE_TESTS_BASE_URI` sets the base URI when downloading firmware for the device-tests
 * `FWUPD_SUPPORTED` overrides the `-Dsupported_build` meson option at runtime
 * `FWUPD_VERBOSE` is set when running `--verbose`
+* `FWUPD_PLUGIN_VERBOSE` can be used to show detailed plugin and backend debugging
```

Comment:
I don't like this name.  It's too similar to just running --verbose for a given plugin.
How about  `FWUPD_BACKEND_VERBOSE`

Reply:
yup, works for me. Will fix when meetings done.

Reply:
Done.